litellm/caching.py

```diff
@@ -1669,13 +1692,14 @@ def delete_cache(self, key):
         if self.redis_cache is not None:
             self.redis_cache.delete_cache(key)
 
+from .redis_gpt_cache import RedisGPTCache
 
 #### LiteLLM.Completion / Embedding Cache ####
 class Cache:
     def __init__(
         self,
         type: Optional[
-            Literal["local", "redis", "redis-semantic", "s3", "disk"]
+            Literal["local", "redis", "redis-semantic", "redis-gptcache", "s3", "disk"]
```

Comment:
Let this name be "gpt_cache_redis". This is what we used in budserve_app. Consistency will be maintained.